.env.development.example

```diff
@@ -1,20 +1,27 @@
 PLASMO_PUBLIC_SHIP_NAME=ncc-1701
 PLASMO_PUBLIC_SHIELD_FREQUENCY=147
 PLASMO_PUBLIC_SITE_URL=https://www.plasmo.com
+CODE=PLASMO-GO-030 # Undefined in extension because it does not start with PLASMO_PUBLIC_
+
+# Extension
+
+PLASMO_PUBLIC_ENV=development
 PLASMO_PUBLIC_EXTENSION_ID=imaadogakjipofgdnmcbeijdmcgedcbp
+CRX_KEY=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0ye66bBwgEeRgJOlE7PtO7jTK9R0ySDlXVUPWPkFt7hDt9hTiu2itexieCW/Sbtc4NCpKJIRVHQUZgr7C2zfZHmC6uLVeb2rba+qzz+5PhwIOP/t7GSpWEJR3MXfrMugQy0pXGBvV3SWFyIdrOhC7HULZT9JVfDcCzS9DGIvZ220VhrBFy41XUuul0AG0RKzuVaeGp/BgD3eIGxedZLFNdth0MKoxqOv8XxvnJ1FPL0jkcmPubJVatxwzI+XdingSQ+BDIOumbBSzXVh/InZbiVNEU0liHCDw/gHsK9EV9frG+hBoVB4V0pXKxbt7aOWXqXzv2WqyDu0oTy+FRvxpQIDAQAB
+
+# Network
+
+PLASMO_PUBLIC_SEPOLIA_ALCHEMY_API_KEY=
+
+# Development Only
+
+PLASMO_PUBLIC_ACCOUNT_CHAIN_ID={1337, 11155111}
 
 PLASMO_PUBLIC_ACCOUNT=0x
 PLASMO_PUBLIC_ACCOUNT_OWNER_PRIVATE_KEY=0x
-PLASMO_PUBLIC_CHAIN_ID=
-PLASMO_PUBLIC_NODE_RPC_URL=
-PLASMO_PUBLIC_BUNDLER_RPC_URL=
 
 PLASMO_PUBLIC_PASSKEY_ACCOUNT=0x
 PLASMO_PUBLIC_PASSKEY_ACCOUNT_CREDENTIAL_ID=
-PLASMO_PUBLIC_PASSKEY_ACCOUNT_FACTORY=0x
 
 PLASMO_PUBLIC_VERIFYING_PAYMASTER=0x
 PLASMO_PUBLIC_VERIFYING_PAYMASTER_OWNER_PRIVATE_KEY=0x
```

Comment:
After testing, these two environment variables require parameters.
Perhaps we can add a `# Requirement` comment before.